### ISSUE TITLE
508-defect-2 [SCREENREADER]: "Add issue" heading should be logically ordered

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25140]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I use headings to help me better understand the structure and hierarchy of a page.

## Details

When adding a new issue on "Step 2 of 4: Issues for Review", there is a `h2` "Add issue" within the card that adds a new issue. This should be a `h3` OR a plain `legend` as it is a child to the `h2` "Step 2 of 4: Issues for Review"

## Acceptance Criteria

- [ ] "Add issue" is a `h3` or a `legend`, not a `h2`

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/14154792/119384747-2f9d8980-bc93-11eb-905f-d6a4c009e1c3.png)

**Comment by Mottie** [comment]
Fix merged into master

**Comment by joshkimux** [comment]
<img width="699" alt="Screen Shot 2021-06-07 at 3 43 45 PM" src="https://user-images.githubusercontent.com/14154792/121078602-29d19900-c7a7-11eb-82f0-0d87d32315e0.png">

- [x] Add issue announces as the group label
- [x] Headings are logical
